Code/Legacy/CrySystem/AZCoreLogSink.h

```diff
@@ -180,6 +180,17 @@ class AZCoreLogSink
             CryLog("(%s) - %s", window, message);
         }
 
+        // If this is an editor-server, then allow the default trace behavior (fwrites to stdout) to occur
+        // The editor will being listening to the stdout of this server
+        if (const auto console = AZ::Interface<AZ::IConsole>::Get())
```

Comment:
nit: It seems like a smell that we have to ask to ask a cvar set in the MultiplayerGem? to check core state around logging. 

Its a little unclear from the comment what returning false does here.

Reply:
I like that idea of not having AZCoreLogSink having to pull a Multiplayer cvar all the time. Check out this new update, I check to see if we're an editor-server inside of SystemInit. A lot of args are used in this area of code, so doesn't seem so strange.

@lumberyard-employee-dm for transparency.